src/hotkey/hotkey_handler.cpp

```diff
@@ -302,6 +307,10 @@ bool play_controller::hotkey_handler::can_execute_command(const hotkey::hotkey_c
 	case hotkey::HOTKEY_SAVE_REPLAY:
 	case hotkey::HOTKEY_LABEL_SETTINGS:
 	case hotkey::LUA_CONSOLE:
+	case hotkey::HOTKEY_SCROLL_UP:
+	case hotkey::HOTKEY_SCROLL_DOWN:
+	case hotkey::HOTKEY_SCROLL_LEFT:
+	case hotkey::HOTKEY_SCROLL_RIGHT:
```

Comment:
I'm wondering if this really belongs in the play_controller; perhaps it should be moved up to the base class (controller_base) instead. As I recall, there is also an editor_controller, and it certainly makes sense to use these keys in the map editor.